### DISCUSSION TITLE
Missing permission `webRequestAuthProvider` in chrome.runtime.Manifest in @types/chrome

### DISCUSSION THREAD

**Discussion by antonshkurenko** [DefinitelyTyped/DefinitelyTyped#69792] [Issues with a @types package]
There is a missing type in here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/chrome/index.d.ts#L7361

In docs: https://developer.chrome.com/docs/extensions/reference/api/webRequest there is a `webRequestAuthProvider`, but not in the lib

[a comment by an automated account is not shown]

**Comment by erwanjugand** [comment]
Relates to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70678
